converter/templates/index.html

```diff
@@ -1,74 +1,327 @@
-            ,function(){$(this).css('box-shadow', '0 0 0 0');
-        });
-        $('#file-box').hover(
-            function(){$(this).css('box-shadow', '0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19)');}
-            ,function(){$(this).css('box-shadow', '0 0 0 0');
+    $(document).ready(function() {
+        $('#text').slideUp(0);
+        $('#file').slideUp(0);
+        $('#result_page').slideUp(0);
+    });
+   
+    $('#text-box').hover(
+        function(){$(this).css('box-shadow', '0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19)');}
+        ,function(){$(this).css('box-shadow', '0 0 0 0');
+    });
+    $('#file-box').hover(
+        function(){$(this).css('box-shadow', '0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19)');}
+        ,function(){$(this).css('box-shadow', '0 0 0 0');
+    });
+   
+    $('#text-button').click(function() {
+        $('#text').slideToggle(300);
+        $('#file').slideUp(300);
+      });
+      
+      $('#file-button').click(function() {
+        $('#file').slideToggle(300);
+        $('#text').slideUp(300);
+      });
+      
+    </script>
+    
+    <script type="text/javascript"> 
+        'use strict';
+        
+        // feature detection for drag&drop upload
+        var isAdvancedUpload = function()
+            {
+                var div = document.createElement( 'div' );
+                return ( ( 'draggable' in div ) || ( 'ondragstart' in div && 'ondrop' in div ) ) && 'FormData' in window && 'FileReader' in window;
+            }();
+
+        // applying the effect for every form
+        var forms = document.querySelectorAll( '.box' );
+        var text_form = document.getElementById('text_input');
+
+        function logSubmit(e) {
+            var form_data = new FormData();
+                form_data.append('text', document.getElementById("textcontent").value);
+                $.ajax({
+                    type: 'POST',
+                    url: '/convert_result/submission',
+                    data: form_data,
+                    contentType: false,
+                    processData: false,
+                    success: function(data) {
+                        var result_text = JSON.parse(data).result;
+                        document.title = "Result Page";
+                        document.getElementById('the_title').innerHTML = "Result";
+                        document.getElementById('result_content').innerHTML = result_text;
+                        if(result_text==="There is something wrong with your input. Please check again!"){
+                            document.getElementById('download').style.display = "none";
+                        }                        
+                        $('#main_page').slideUp(500);
+                        $('#result_page').slideDown(1500);
+                    },
+                    error: function(data){
+                        window.location.href = '/';
```

Comment:
Need to handle error properly instead of just redirecting page. Should indicate an error somewhere in the page.

Reply:
error message is sent through flash from a python script, this line is used to refresh the page to show the flash message